long-ride-alerts/src/main/java/org/apache/flink/training/exercises/longrides/LongRidesExercise.java

```diff
@@ -98,17 +100,71 @@ public static void main(String[] args) throws Exception {
     @VisibleForTesting
     public static class AlertFunction extends KeyedProcessFunction<Long, TaxiRide, Long> {
 
+
+        private transient ValueState<TaxiRide> rideState;
+
+        private final int TWO_HOURS = 2 * 60 * 60;
```

Comment:
I see it as a
`private static final int TWO_HOURS = 2 * 60 * 60;`
(static)
and in this case it should be declared before just any private fields.

Reply:
@ness-DianaScurtu please pay attention to details.
It has to be `private static final` not just `private final`